scripts/inner/kubestellar-kube-bind

```diff
@@ -56,15 +71,28 @@ then
     exit 1
 fi
 
-conf_file="/tmp/$space_name.conf"
-kubectl-kubestellar-get-config-for-space --space-name $space_name --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --in-cluster --output $conf_file
-KUBECONFIG=$conf_file
+# Some place protectable to store kubeconfig files.
+# We do not have promises about what the $PWD is.
+# Some users administer multiple KubeStellar systems,
+# so the name must not be a pure function of the space name.
+konnect_dir="${HOME}/kubestellar-konnectors/$(date +%y%m%d-%H%M%S)-${space_name}-${resource_to_bind}-$$"
```

Comment:
Adding resource_to_bind is not reasonable. The konnector is using a config matching a specific space, and is used for all resources bounded into that space.  
Also, the konnector process is tagged and validated using env variable  that uses only the space name so a new KS instance using the same space name will mistakenly  avoid creating a konnector .

Reply:
In the case of the Helm chart there is a fixed usage pattern, this is easier.

I was thinking also of the bare processes case. But I forgot that bare process deployment is only for contributors doing development and testing --- not for users administering possibly multiple KubeStellar deployments. So it _is_ easier than I was thinking.

I am going to revise this further to push responsibility up to the caller (kubectl kubestellar ensure wmw and is).

Reply:
On second thought, I am not going to move that responsibility up the stack, because the mailbox-controller is not currently dealing in kubeconfig files for the MBSes.